apps/db/prisma/schema.prisma

```diff
@@ -28,7 +28,7 @@ model Session {
   id Int @id @default(autoincrement())
```

Comment:
I did some research, [https://thecopenhagenbook.com/sessions](https://thecopenhagenbook.com/sessions), let's change this schema a bit.

We should delete the `id` field and then rename `token` to `id` based on how we are using it.